apps/api/v2/src/modules/users/inputs/get-users.input.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsOptional, Validate } from "class-validator";
+
+import { IsEmailStringOrArray } from "../validators/isEmailStringOrArray";
+
+export class GetUsersInput {
+  @IsOptional()
+  @Validate(IsEmailStringOrArray)
+  @ApiProperty({
+    description: "The email address or an array of email addresses to filter by",
+  })
+  email?: string | string[];
+}
```

Comment:
here we should transform the value to an array if it is a string

Reply:
@Transform(({ value }: { value: string | string[] }) => {
    return typeof value === "string" ? [value] : value;
  })
email?: string[];